docs/03-tutorials/00-eventing/evnt-02-subs-with-multiple-filters.md

```diff
@@ -49,24 +50,10 @@ cat <<EOF | kubectl apply -f -
       namespace: default
     spec:
       sink: 'http://lastorder.default.svc.cluster.local'
-      filter:
-        filters:
-          - eventSource:
-              property: source
-              type: exact
-              value: ''
-            eventType:
-              property: type
-              type: exact
-              value: sap.kyma.custom.myapp.order.received.v1
-          - eventSource:
-              property: source
-              type: exact
-              value: ''
-            eventType:
-              property: type
-              type: exact
-              value: sap.kyma.custom.myapp.order.changed.v1
+      source: myapp
```

Comment:
don't forget to change the `apiVersion` to `eventing.kyma-project.io/v1alpha2`.

Reply:
Updated.